src/index.ts

```diff
@@ -18,3 +18,4 @@ export * as stark from './utils/stark';
 export * as ec from './utils/ellipticCurve';
 export * as uint256 from './utils/uint256';
 export * as shortString from './utils/shortString';
+export * as eip712 from './utils/eip712';
```

Comment:
We should not make references to EIP 712 in the code as we are not following the standard exactly. The current specification is only inspired by the EIP 712 standard.